src/plugins/vis_types/timelion/server/series_functions/abs.test.js

```diff
@@ -11,7 +11,7 @@ import fn from './abs';
 import _ from 'lodash';
 import expect from '@kbn/expect';
 const seriesList = require('./fixtures/series_list')();
-import invoke from './helpers/invoke_series_fn';
+import invoke from './test_helpers/invoke_series_fn';
```

Comment:
`helpers` is a pretty ambiguous name, so I renamed the directory to `test_helpers` to be more clear about the purpose of the code and allow the classifier to assert that this is test code which shouldn't be imported by non-test code.